WalletWasabi/Models/UnconfirmedTransactionChainItemLegacy.cs

```diff
@@ -6,7 +6,9 @@
 using WalletWasabi.JsonConverters.Collections;
 
 namespace WalletWasabi.Models;
-public record UnconfirmedTransactionChainItem(
+
+// This class is here for backward compatibility and should be removed on removal of unconfirmed-transaction-chain endpoint.
+public record UnconfirmedTransactionChainItemLegacy(
```

Comment:
I would undo this cahnage. No change should be done on the server code because the change is clearly a client-side code only.